platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/migration/SystemConfigurationMigrationTest.java

```diff
@@ -516,6 +557,14 @@ private Path createTempDdf() throws IOException {
         Files.createFile(rootTempDir.resolve(INVALID_KEYSTORE_REL_PATH));
         Files.createFile(rootTempDir.resolve(INVALID_TRUSTSTORE_REL_PATH));
         Files.createFile(rootTempDir.resolve(INVALID_CRL_REL_PATH));
+
+        existingFilePath = Paths.get(ddfHome.resolve("orig.config")
```

Comment:
i need the existingFilePath in creating the symbolic link. If I didn't have it saved, then I wouldve needed to type out the path again when I create the symbolic link